### ISSUE TITLE
 Approximate Lipschitz Loss Learner

### ISSUE THREAD

**Issue by lakshmiDRIP** [lakshmiDRIP/DROP#431]
<i>ApproximateLipschitzLossLearner</i> implements the Learner Class that holds the Space of Normed R<sup>d</sup> To Normed R<sup>1</sup> Learning Functions for the Family of Loss Functions that are "approximately" Lipschitz, i.e.,
 
 				loss (ep) - loss (ep') Less Than max (C * |ep-ep'|, C')

 The References are:
  
 * Alon, N., S. Ben-David, N. Cesa Bianchi, and D. Haussler (1997): Scale-sensitive Dimensions, Uniform Convergence, and Learnability <i>Journal of Association of Computational Machinery</i> <b>44 (4)</b> 615-631

 * Anthony, M., and P. L. Bartlett (1999): <i>Artificial Neural Network Learning - Theoretical Foundations</i> <b>Cambridge University Press</b> Cambridge, UK

 * Kearns, M. J., R. E. Schapire, and L. M. Sellie (1994): <i>Towards Efficient Agnostic Learning</i> Machine Learning <b>17 (2)</b> 115-141

 * Lee, W. S., P. L. Bartlett, and R. C. Williamson (1998): The Importance of Convexity in Learning with Squared Loss <i>IEEE Transactions on Information Theory</i> <b>44</b> 1974-1980

 * Vapnik, V. N. (1998): <i>Statistical learning Theory</i> <b>Wiley</b> New York
